src/tools/registry/gen_inc.c

```diff
@@ -1386,10 +1386,10 @@ int parse_var_array(FILE *fd, ezxml_t registry, ezxml_t superStruct, ezxml_t var
 				fortprintf(fd, "         call mpas_add_att(%s(%d) %% attLists(const_index) %% attList, 'units', '%s')\n", pointer_name, time_lev, temp_str);
 			}
 
-			if ( vararrmissingval ) {
-				fortprintf(fd, "         call mpas_add_att(%s(%d) %% attLists(const_index) %% attList, 'missing_value', %s)\n", pointer_name, time_lev, missing_value);
+			if ( vararrmissingval != NULL ) {
+				// fortprintf(fd, "         call mpas_add_att(%s(%d) %% attLists(const_index) %% attList, 'missing_value', %s)\n", pointer_name, time_lev, missing_value);
 				// Uncomment to add _FillValue to match missing_value
-				// fortprintf(fd, "         call mpas_add_att(%s(%d) %% attLists(const_index) %% attList, '_FillValue', %s)\n", pointer_name, time_lev, missing_value);
+				fortprintf(fd, "         call mpas_add_att(%s(%d) %% attLists(const_index) %% attList, '_FillValue', %s)\n", pointer_name, time_lev, missing_value);
```

Comment:
Since this is a framework file, we need to merge this into the `develop` branch instead, and then I will merge `develop` into the LANL branches: `ocean/develop`, `seaice/develop`, `landice/develop`.  @wenshanw, don't worry about that right now. This PR is convenient for testing, and I can help with the different branches after testing.

Reply:
Right, I was thinking that would be the right approach, too, and that @wenshanw shouldn't have to worry about that.  We need to do more testing anyway.